Misc/NEWS.d/next/Windows/2024-12-16-16-16-35.gh-issue-127951.lpE13-.rst

```diff
@@ -0,0 +1 @@
+Add option `--pystats` to the Windows build to enable performance statistics collection.
```

Comment:
```suggestion
Add option ``--pystats`` to the Windows build to enable performance statistics collection.
```

Double quotes are needed in RST format.

Also, could you move this to the `Build` directory instead of `Windows`?